src/index.js

```diff
@@ -462,7 +462,7 @@ class Ooth {
             })
 
             if (this.onLogin) {
-                this.onLogin(profile)
+                await this.onLogin(profile)
```

Comment:
This is a fork and I'm not sure how important this is, but since we don't know for sure that every existing implementation of `this.onLogin` is set up to return a promise, the safer thing to do here is probably `await Promise.resolve(this.onLogin(profile))`. That will guarantee that it would work even if `this.onLogin()` doesn't return a promise.